features/svc-catalog_asb/svc-catalog.feature

```diff
@@ -207,6 +207,152 @@ Feature: Service-catalog related scenarios
       | o        | custom-columns=CLASSNAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName |
     Then the output should not contain "user-provided"
 
+  # @author chezhang@redhat.com
+  # @case_id OCP-15603
+  @admin
+  @destructive
+  Scenario: Create/get/update/delete for ServiceInstance resource
+    Given I have a project
+    And evaluation of `project.name` is stored in the :ups_broker_project clipboard
+    And I create a new project
+    And evaluation of `project.name` is stored in the :user_project clipboard
+
+    # Deploy ups broker
+    Given I register clean-up steps:
+    """
+    I run the :delete admin command with:
```

Comment:
@emmajiafan I think we can replace this with `admin ensure...`

Reply:
Updated